app/views/govuk_component/docs/govspeak.yml

```diff
@@ -121,15 +121,12 @@ fixtures:
         <p>My quote</p>
         <p class="last-child">about things</p>
       </blockquote>
-  call_out_blocks:
+  information_block:
```

Comment:
As `.information-block` was removed in https://github.com/alphagov/static/pull/899/commits/943d1e1437b8625c3295dd0c1a4ef27d873471d1, can the fixture be removed now?